pkg/views/workspace/create/configuration.go

```diff
@@ -19,13 +19,15 @@ import (
 var configurationHelpLine = lipgloss.NewStyle().Foreground(views.Gray).Render("enter: next  f10: advanced configuration")
 
 type ProjectConfigurationData struct {
-	Image             string
-	User              string
-	PostStartCommands []string
-	EnvVars           map[string]string
+	BuilderChoice        string
```

Comment:
`BuilderChoice` enum?

Reply:
I would leave it as is for now. We can discuss this when the builder specs are complete.